src/main/scala/com.snowplowanalytics.sauna/responders/optimizely/DcpResponder.scala

```diff
@@ -66,15 +66,19 @@ class DcpResponder(optimizely: Optimizely, importRegion: String, val logger: Act
    * @return Some [[CustomersProfilesPublished]] if this observer-event need to be
    *         processed by this responder, None if event need to be skept
    */
-  def extractEvent(observerEvent: ObserverBatchEvent): Option[CustomersProfilesPublished] = {
-    extractCustomerPublished(observerEvent) match {
-      case Right(event) => Some(event)
-      case Left(Some(error)) =>
-        // TODO: cases where path is only partly correct should be handled differently, without deleting source file
-        notify(error)
-        None
-      case Left(None) =>
-        None
+  def extractEvent(observerEvent: ObserverEvent): Option[CustomersProfilesPublished] = {
+    observerEvent match {
+      case e: ObserverFileEvent =>
+        extractCustomerPublished(e) match {
+          case Right(event) => Some(event)
+          case Left(Some(error)) =>
+            // TODO: cases where path is only partly correct should be handled differently, without deleting source file
```

Comment:
This is a note from the 0.1.0 release - not quite sure what it refers to. I'll look into this later.